### ISSUE TITLE
C2ChoiceStyle borderColor is ignored by C2Chip.build()

### ISSUE THREAD

**Issue by slavap** [davigmacode/flutter_chips_choice#37]
Currently:
```dart
final Color effectiveBorderColor = data.selected
        ? secondaryColor.withOpacity(effectiveBorderOpacity)
        : primaryColor.withOpacity(effectiveBorderOpacity);
```
Where is style?.borderColor processing?
So the following code is doing nothing:
```dart
choiceStyle: C2ChoiceStyle(borderColor: Colors.red, borderWidth: 2, borderOpacity: 0.5);
```

**Comment by davigmacode** [comment]
Fixed in `^2.1.0` and you still need to set `appearance: C2ChipType.outlined,` to `C2ChoiceStyle`, if this still not enough, you can use `choiceBuilder` option to create a custom chip, here is the [example](https://github.com/davigmacode/flutter_chips_choice/blob/master/example/lib/main.dart#L532)

I hope find a better and flexible way to styling the chip widget

**Comment by slavap** [comment]
@davigmacode Thank you! it works now.
